api/controller.js

```diff
@@ -2,10 +2,7 @@
 
 var properties = require('../package.json')
 var distance = require('../service/distance');
```

Comment:
Do we still need the distance service? What is its purpose?

Reply:
Probably not, David put it there for test purpose i believe.